components/NavLandingpage/index.js

```diff
@@ -4,86 +4,93 @@ import Image from "next/image";
-      </StyledLink>
-      <StyledLink href={"/medicine"}>
-        <StyledIcon
-          src={"/icons/medicine.png"}
-          alt={"Medizin-Icon"}
-          width={100}
-          height={60}
-        />
-        <StyledSpan>Medizin</StyledSpan>
-      </StyledLink>
-      <StyledLink href={"/favorites"}>
-        <StyledIcon
-          src={"/icons/heart.png"}
-          alt={"Favoriten-Icon"}
-          width={100}
-          height={60}
-        />
-        <StyledSpan>meine Favoriten</StyledSpan>
-      </StyledLink>
-      <StyledLink href={"/add"}>
-        <StyledIcon
-          src={"/icons/add.png"}
-          alt={"Favoriten-Icon"}
-          width={100}
-          height={60}
-        />
-        <StyledSpan>Einrichtung hinzufügen</StyledSpan>
-      </StyledLink>
-    </StyledNav>
+    <>
+      <StyledNav>
+        <StyledNavLink href={"/sleep"}>
+          <StyledIcon
+            src={"/assets/svg/sleep.svg"}
+            alt={"Schlafen-Icon"}
+            width={100}
+            height={60}
+          />
+          <StyledSpan>Schlafen</StyledSpan>
+        </StyledNavLink>
+        <StyledNavLink href={"/hygiene"}>
+          <StyledIcon
+            src={"/assets/svg/hygiene.svg"}
+            alt={"Hygiene-Icon"}
+            width={100}
+            height={60}
+          />
+          <StyledSpan>Hygiene</StyledSpan>
+        </StyledNavLink>
+        <StyledNavLink href={"/medicine"}>
+          <StyledIcon
+            src={"/assets/svg/medicine.svg"}
+            alt={"Medizin-Icon"}
+            width={100}
+            height={60}
+          />
+          <StyledSpan>Medizin</StyledSpan>
+        </StyledNavLink>
+        <StyledNavLink href={"/favorites"}>
+          <StyledIcon
+            src={"/assets/svg/favorites.svg"}
+            alt={"Favoriten-Icon"}
+            width={100}
+            height={60}
+          />
+          <StyledSpan>Favoriten</StyledSpan>
+        </StyledNavLink>
+      </StyledNav>
+
+      <StyledLinkContainer>
+        <StyledLink href={"/add"}>Einrichtung hinzufügen</StyledLink>
+      </StyledLinkContainer>
+    </>
   );
 }
 
-const StyledLink = styled(Link)`
-  border-style: solid;
+const StyledNavLink = styled(Link)`
+  border-radius: 40px;
+  background-color: var(--secondary-color);
+  box-shadow: 0 2px 10px 0 rgba(0, 0, 0, 0.1);
+  // margin: 0 10px 10px 10px;
   text-decoration: none;
-  width: 120px;
-  height: 120px;
-  display: flex;
-  margin-top: 15px;
-  border-radius: 10px;
-  box-shadow: rgba(50, 50, 93, 0.25) 0px 25px 80px -25px;
-  color: #0000ee;
+  text-align: center;
+  padding-bottom: 40px;
 `;
 
 const StyledSpan = styled.span`
-  margin-top: 65px;
-  position: absolute;
-  margin-left: 5px;
-  padding: 10px;
-  color: black;
-  max-width: 120px;
+  color: var(--font-color);
+  font-size: 18px;
+  font-weight: bold;
+  letter-spacing: 0;
+  line-height: 24px;
+  width: 100%;
+  margin-top: 10px;
+  margin-bottom: 40px;
 `;
```

Comment:
Test the styled in the dev tools and you will see, that you can remove the most of them 🥳

```suggestion
  color: var(--font-color);
  font-size: 1.2rem;
`;
```